python/hsfs/core/job.py

```diff
@@ -58,3 +59,6 @@ def executions(self):
     @property
     def href(self):
         return self._href
+
+    def run(self, await_termination=True):
+        engine.get_instance().run_job(self, await_termination)
```

Comment:
see other comment, one more suggestion
```suggestion
        engine.get_instance().run_job(self, {"wait_for_job": await_termination})
```